src/BenchmarksDriver/Program.cs

```diff
@@ -1188,7 +1184,8 @@ public static int Main(string[] args)
         private static async Task<ClientJob> RunClientJob(string scenarioName, Uri clientUri, Uri serverJobUri, string serverBenchmarkUri)
         {
             var clientJob = new ClientJob(_clientJob) { ServerBenchmarkUri = serverBenchmarkUri };
-
+            var benchmarkUri = new Uri(serverBenchmarkUri);
+            clientJob.Headers["Host"] = benchmarkUri.Host + ":" + benchmarkUri.Port;
```

Comment:
IIS validates host header

Reply:
I believe TechEmpower uses `Host: TFB-server`:

https://github.com/TechEmpower/FrameworkBenchmarks/search?utf8=%E2%9C%93&q=server_host&type=

Is there any reason to think adding `:port` would make the server slower?  If so, we should measure the real-world impact of changing this header before merging.

Reply:
FYI, related to https://github.com/aspnet/benchmarks/issues/381

Reply:
I asked @halter73 about this, he said that is shouldn't affect results.

Reply:
After @Tratcher changes, Kestrel will validate this a little differently than it did for just "localhost" (e.g. Kestrel will call HttpUtilities.IsNumeric(portChar)), but I can't imagine that having a measurable impact on these benchmarks.